tools/create_operators_pull_request.py

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import logging
+import os.path
+import sys
+import tempfile
+
+from utils import (
+    run,
+    authenticated_github_repository_url,
+    clone_repository,
+    random_short_string,
+    get_git_user,
+    configure_git_user,
+    create_pull_request,
+)
+
+PROGRAM_NAME = os.path.basename(__file__)
+
+log = logging.getLogger(__name__)
+
+
+OPERATORS_REPOSITORY = "kudobuilder/operators"
+
+
+def parse_arguments() -> dict:
+    parser = argparse.ArgumentParser(
+        description="Open a PR copying a KUDO Operator's files to the "
+        + f"{OPERATORS_REPOSITORY} repository"
+    )
+
+    parser.add_argument(
+        "--operator-repository",
+        type=str,
+        required=True,
+        help="The KUDO Operator repository to copy the files from "
+        + "(e.g., mesosphere/kudo-cassandra-operator)",
+    )
+    parser.add_argument(
+        "--operator-name",
+        type=str,
+        required=True,
+        help="The name of the KUDO operator (e.g., cassandra, kafka, spark)",
+    )
+    parser.add_argument(
+        "--operator-git-tag",
+        type=str,
+        required=True,
+        help="The git tag in the KUDO operator repository to copy the files "
+        + f"from. This is also the directory name in {OPERATORS_REPOSITORY}, "
+        + "e.g., repository/cassandra/$OPERATOR_GIT_TAG",
+    )
+    parser.add_argument(
+        "--github-token",
+        type=str,
+        required=True,
+        help="The GitHub token used for opening the PR against "
+        + f"{OPERATORS_REPOSITORY}",
+    )
+    parser.add_argument(
+        "--git-commit-message",
+        type=str,
+        help=f"The git commit message in the {OPERATORS_REPOSITORY} PR branch. "
+        + "This will also be the PR title",
+    )
+    parser.add_argument(
+        "--operators-base-branch",
+        type=str,
+        default="master",
+        help=f"The {OPERATORS_REPOSITORY} branch to open a PR against",
+    )
+    parser.add_argument(
+        "--operators-repository",
+        type=str,
+        default="master",
```

Comment:
This default seems wrong.

Reply:
Thanks! d4dc22b987ab52568e9ffae0731b8c0bb39433ab